cmd/stackdriver-prometheus-sidecar/main.go

```diff
@@ -203,6 +207,9 @@ func main() {
 	a.Flag("stackdriver.api-address", "Address of the Stackdriver Monitoring API.").
 		Default("https://monitoring.googleapis.com:443/").URLVar(&cfg.stackdriverAddress)
 
+	a.Flag("stackdriver.api-override", "List of IP addresses. If not empty, stackdriver.api-address will always resolve to these addresses.").
```

Comment:
This comment is no longer accurate, because this flag is now just a bool. Can you fix it?

I suggest:
- Change this flag to `stackdriver.use-restricted-ips`
- Rename the bool variable to `useRestrictedIps`
- Update the flag's comment to "If true, send all requests through restricted VIPs (EXPERIMENTAL)"
The comment is a bit obscure, but it won't be there for very long.

Reply:
Makes sense, thanks. Doing now.